### ISSUE TITLE
Multiple instances of Content Tabs break the fixed panel width

### ISSUE THREAD

**Issue by antonymayi** [jbms/sphinx-immaterial#108]
When using more than one instance of content tabs and having one of the tab contents exceeding the panel width, it breaks the layout as everything gets expanded to the exceeded width.

Example document:

```
FooBarBaz
=========

.. md-tab-set::

    .. md-tab-item:: Foo

        .. code-block:: python

            # THIS IS A VERY LARGE LINE EXCEEDING THE FIXED WIDTH OF THE NAV PANEL CAUSING IT TO BREAK THE LAYOUT FOR THE REST OF THE PAGE


    .. md-tab-item:: Bar

        .. code-block:: python

            # BAR

Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna
aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.


.. md-tab-set::

    .. md-tab-item:: Foo

        .. code-block:: python

            # THIS IS A VERY LARGE LINE EXCEEDING THE FIXED WIDTH OF THE NAV PANEL CAUSING IT TO BREAK THE LAYOUT FOR THE REST OF THE PAGE


    .. md-tab-item:: Bar

        .. code-block:: python

            # THIS IS A SHORT LINE - NO ISSUES CAUSED

```

This will render the following (broken) layout when the *Foo* tab (with exceeding content) is selected:

![navfoo](https://user-images.githubusercontent.com/6304370/171687671-f6b589a8-8b25-40b6-9abd-193688f70721.jpg)

While the following (no issues) when the *Bar* tab (with narrow content) is selected:
![navbar](https://user-images.githubusercontent.com/6304370/171687870-682b0fa6-3c4e-4088-97df-96cfa9fad549.jpg)

**Comment by jbms** [comment]
Can you check if you can reproduce this in mkdocs-material?

**Comment by 2bndy5** [comment]
That is very strange. I suspect a CSS problem as the HTML for code blocks produced in Sphinx doesn't exactly match the HTML for code blocks produced by the MD ext that's used in mkdocs.

**Comment by 2bndy5** [comment]
scratch my previous comment. I took a look at the rendering of the snippet in OP and I found that the first set of tabs isn't properly closed with a `</div>` tag. I've seen this before when developing the `.. rst-result::` directive, but I wasn't able to determine the cause..

<details><summary>Full HTML of rendered snippet</summary>

```html
<h2 id="foobarbaz">FooBarBaz<a class="headerlink" href="#foobarbaz" title="Permalink to this headline">¶</a></h2>
<div class="tabbed-set tabbed-alternate" data-tabs="1:2" style="--md-indicator-x: 0px; --md-indicator-width: 53px;">
  <input checked="" type="radio" id="__tabbed_1_1" name="__tabbed_1"><input type="radio" id="__tabbed_1_2"
    name="__tabbed_1">
  <div class="tabbed-labels docutils container">
    <label class="tabbed-label" for="__tabbed_1_1">
      Foo</label><label class="tabbed-label" for="__tabbed_1_2">
      Bar</label></div>
  <div class="tabbed-content docutils container">
    <div class="tabbed-block docutils container">
      <div class="highlight-python notranslate">
        <div class="highlight">
          <pre id="__code_2"><span></span><button class="md-clipboard md-icon" title="Copy to clipboard" data-clipboard-target="#__code_2 > code"></button><code tabindex="0"><span class="c1"># THIS IS A VERY LARGE LINE EXCEEDING THE FIXED WIDTH OF THE NAV PANEL CAUSING IT TO BREAK THE LAYOUT FOR THE REST OF THE PAGE</span>
</code></pre>
        </div>
      </div>
    </div>
    <div class="tabbed-block docutils container">
      <div class="highlight-python notranslate">
        <div class="highlight">
          <pre><span></span><code><span class="c1"># BAR</span>
</code></pre>
        </div>
      </div>
    </div>
  </div>
  <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna
    aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
    Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur
    sint
    occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
  <div class="tabbed-set tabbed-alternate" data-tabs="2:2" style="--md-indicator-x: 0px; --md-indicator-width: 53px;">
    <input checked="" type="radio" id="__tabbed_2_1" name="__tabbed_2"><input type="radio" id="__tabbed_2_2"
      name="__tabbed_2">
    <div class="tabbed-labels docutils container">
      <label class="tabbed-label" for="__tabbed_2_1">
        Foo</label><label class="tabbed-label" for="__tabbed_2_2">
        Bar</label></div>
    <div class="tabbed-content docutils container">
      <div class="tabbed-block docutils container">
        <div class="highlight-python notranslate">
          <div class="highlight">
            <pre id="__code_3"><span></span><button class="md-clipboard md-icon" title="Copy to clipboard" data-clipboard-target="#__code_3 > code"></button><code><span class="c1"># THIS IS A VERY LARGE LINE EXCEEDING THE FIXED WIDTH OF THE NAV PANEL CAUSING IT TO BREAK THE LAYOUT FOR THE REST OF THE PAGE</span>
</code></pre>
          </div>
        </div>
      </div>
      <div class="tabbed-block docutils container">
        <div class="highlight-python notranslate">
          <div class="highlight">
            <pre><span></span><code><span class="c1"># THIS IS A SHORT LINE - NO ISSUES CAUSED</span>
</code></pre>
          </div>
        </div>
      </div>
    </div>



    <hr>
    <div class="md-source-file">
      <small>

        Last update:
        Jun 02, 2022

      </small>
    </div>


  </div>
</div>
```
</details>

**Comment by 2bndy5** [comment]
I think this is caused by raising `SkipNode` here:
https://github.com/jbms/sphinx-immaterial/blob/b66c9efe77bde316491279bd114f83d8f43b31c9/sphinx_immaterial/content_tabs.py#L170

**Comment by 2bndy5** [comment]
should be fixed in v0.7.3
